en/docs/design/rate-limiting/throttle-policy-import-export.md

```diff
@@ -0,0 +1,323 @@
+# Importing and Exporting Throttle Policies
+
+Throttling policies exist at different levels such as Application Level, Subscriber Level, Application Level, Advanced Policy Level, etc. These throttling policies can belong to different environments according to the product lifecycle and are related to different APIs or Applications. Of these throttling policies, some of them exist as default policies belonging to every environment, and throttling policies can be custom-made. Users might need to import custom-made or default throttling policies from one environment to another.
+
+When an API or an Application is imported from an environment that has custom-made throttling policies to another environment, if the API or the  Application has subscribed to a throttling policy that is not in the destination environment, an error is thrown indicating the throttling policy is not supported and the importation fails.
```

Comment:
Good to reword this. Instead of saying an error is thrown, we can say `... if the API or the  Application has subscribed to a throttling policy that is not in the destination environment, importation fails indicating the throttling policy is not supported.`

Reply:
```suggestion
You may encounter an error when an API or an Application is imported from an environment that has custom-made throttling policies to another environment. This happens if the API or the Application has subscribed to a throttling policy that is not in the destination environment. Importation fails to indicate the throttling policy is not supported.
```